transformation/src/jade-schema/tables/experiment_activity.table.json

```
@@ -0,0 +1,88 @@
+{
+  "name": "experiment_activity",
+  "columns": [
+    {
+      "name": "id",
+      "datatype": "string",
+      "type": "primary_key"
+    },
+    {
+      "name": "time_created",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_started",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_ended",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "time_submitted",
+      "datatype": "timestamp",
+      "type": "required"
+    },
+    {
+      "name": "description",
+      "datatype": "string",
+      "type": "required"
+    },
+    {
+      "name": "assay",
```

```suggestion
      "name": "assay_id",
```
? Or is it not a FK?

---

Kathy suggested that we actually just get rid of this column.